app/retail/templates/email_settings.html

```diff
@@ -61,6 +61,20 @@ <h5>{% trans "Basic Preferences" %}</h5>
                             <small id="level" class="form__input-help">{% trans "roundup emails once a week" %}</small>
                             <small id="level" class="form__input-help">{% trans "notifications up to twice a day" %}</small>
                         </div>
+                        <div class="form-group">
+                            <label class="form__label" for="level">{% trans "Preferred language" %}</label>
+                            <div class="form__select">
+                                <select id="preferred_language" name="preferred_language" required>
+                                    {% get_language_info_list for available_languages as langs %}
```

Comment:
You should be able to replace this with:

```html
{% get_available_languages as global_languages %}
{% get_language_info_list for global_languages as langs %}
```
Then you can eliminate the `'available_languages': ['en', 'pl'],` part all together.